### PR TITLE
Repair SMC funi stations

### PR DESCRIPTION
**Pull request by ConscritNeuneu**

### CODE REVIEW

File: stations.csv [hunk 1]
```diff
@@ -4571,7 +4571,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 6062;Les Avants;les-avants;8501377;;46.453807;6.943331;;CH;Europe/Zurich;f;f;f;f;f;f;CHABZ;;f;;f;8501377;f;;f;;f;;f;8501377;t;;f;8501377;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 6064;Montbovon;montbovon;8504070;;46.486743;7.04285;;CH;Europe/Zurich;f;f;f;t;f;f;CHACB;;f;;f;8504070;f;;f;;f;;f;8504070;t;;f;8504070;f;;f;;f;;;;f;;f;;f;;f;CHACB;t;;f;f;;;;;;;;;;;;;;;;;;;
 6070;Solothurn;solothurn;8500207;;47.2041720000;7.5426860000;;CH;Europe/Zurich;f;f;f;t;f;f;CHACH;;f;;f;8500207;t;;f;CHZKS;t;19348;t;8500207;t;;f;8500207;f;;f;;f;;;;f;;f;;f;;f;CHACH;t;;f;f;;;;Soleura;Soleure;Soletta;;;;;;ゾロトゥルン;졸로투른;Solura;Soleura;Золотурн;;;索洛图恩
-6071;Montana vermala;montana-vermala;8501596;;;;;CH;Europe/Zurich;f;f;f;f;f;f;CHACI;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
+6071;Crans Montana Gare;crans-montana-gare;8501596;;46.312905;7.489221;;CH;Europe/Zurich;f;f;f;t;f;f;CHACI;;f;;f;8501596;f;;f;;f;;f;8501596;t;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
```
Comment (ConscritNeuneu) [inline stations.csv]: is_suggestable → true

File: stations.csv [hunk 2]
```diff
@@ -1693,7 +1693,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 2236;Prin-Deyrançon;prin-deyrancon;8748526;87485268;46.2190177;-0.6365007;;FR;Europe/Paris;f;f;f;t;f;f;FREIR;PND;t;;f;8701440;f;;f;;f;;f;;f;;f;8701440;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 2237;Épannes;epannes;8748527;;46.2333330000;-0.5833330000;;FR;Europe/Paris;f;f;f;f;f;f;FREIS;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 2238;Fors;fors;8748532;87485326;46.2345203;-0.4069449;;FR;Europe/Paris;f;f;f;t;f;f;FREIV;FOW;t;;f;8703339;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
-2239;Marigny;marigny;8748533;87485334;46.1990832;-0.428404;;FR;Europe/Paris;f;f;f;t;f;f;FREIW;MRJ;t;;f;8703911;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
+2239;Marigny;marigny;8748533;87485334;46.1990832;-0.428404;;FR;Europe/Paris;f;f;f;t;t;f;FREIW;MRJ;t;;f;8703911;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
```
Comment (ConscritNeuneu) [inline stations.csv]: country_hint → true

File: stations.csv [hunk 3]
```diff
@@ -16273,8 +16272,8 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
...
 18207;Leysin TLSA;leysin-tlsa;;;46.3475090000;7.0173840000;;CH;Europe/Zurich;f;f;f;f;f;f;;;f;;f;8501467;t;;f;;f;;f;8501467;t;;f;8501467;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 18208;Sion;sion;8501506;;46.2274606182505;7.359364628791808;;CH;Europe/Zurich;f;f;f;t;f;f;CHSIR;;t;;f;8501506;t;u0jnzr;t;CHSIR;t;19338;t;8501506;t;;f;8501506;f;;f;8501506;t;S01225;;;f;;f;8500585;f;;f;CHSIR;t;;f;f;;Sitten;;;;;;;;;;シオン;시옹;;Sião;Сьон;;;锡永
-18209;Sierre/Siders;sierre-siders;8501509;;46.2921180000;7.5328070000;;CH;Europe/Zurich;f;f;f;t;f;f;CHZKO;;t;;f;8501509;t;;f;;f;;f;8501509;t;;f;;f;;f;;f;;;;f;;f;8500242;f;;f;CHZKO;t;;f;f;;;;;;;;;;;;;;;;;;;
-18210;Sierre/Siders SMC;sierre-siders-smc;;;46.2927830000;7.5299130000;;CH;Europe/Zurich;f;f;f;f;f;f;;;f;;f;8501590;t;;f;;f;;f;8501590;t;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
+18209;Sierre/Siders;sierre-siders;8501509;;46.2921180000;7.5328070000;;CH;Europe/Zurich;f;t;f;t;f;f;CHZKO;;f;;f;8501509;t;;f;;f;;f;8501509;t;;f;;f;;f;;f;;;;f;;f;8500242;f;;f;CHZKO;t;;f;f;;;;;;;;;;;;;;;;;;;
```
Comment (ConscritNeuneu) [inline stations.csv]: is_main_station → true
sncf_is_enabled → false

File: stations.csv [hunk 4]
```diff
@@ -16273,8 +16272,8 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
...
+18210;Sierre/Siders SMC;sierre-siders-smc;8501590;;46.2927830000;7.5299130000;18209;CH;Europe/Zurich;f;f;f;f;f;f;;;f;;f;8501590;t;;f;;f;;f;8501590;t;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
```
Comment (ConscritNeuneu) [inline stations.csv]: added parent_station_id

File: stations.csv [hunk 5]
```diff
@@ -16286,7 +16285,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 18219;Zermatt GGB;zermatt-ggb;8501690;;46.0237180000;7.7495010000;;CH;Europe/Zurich;f;f;f;f;f;f;;;f;;f;8501690;t;;f;;f;;f;8501690;t;;f;8501690;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 18220;Leysin-Feydey gare;leysin-feydey-gare;;;46.3447320000;7.0089430000;;CH;Europe/Zurich;f;f;f;f;f;f;;;f;;f;8501974;t;;f;;f;;f;8501974;t;;f;8501974;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 18221;Sion poste/gare;sion-poste-gare;;;46.2278270000;7.3582550000;;CH;Europe/Zurich;f;f;f;f;f;f;;;f;;f;8501994;t;;f;;f;;f;8501994;t;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
-18222;Sierre poste/gare;sierre-poste-gare;;;46.2922440000;7.5326900000;;CH;Europe/Zurich;f;f;f;f;f;f;;;f;;f;8501996;t;;f;;f;;f;8501996;t;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
+18222;Sierre poste/gare;sierre-poste-gare;;;46.2922440000;7.5326900000;18209;CH;Europe/Zurich;f;f;f;f;f;f;;;f;;f;8501996;t;;f;;f;;f;8501996;t;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
```
Comment (ConscritNeuneu) [inline stations.csv]: added parent_station_id

File: stations.csv [hunk 6]
```diff
@@ -30546,7 +30545,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 33370;Linstow;linstow;;;53.618935;12.3712165;;DE;Europe/Berlin;t;f;f;t;f;f;;;f;;f;;f;;f;;f;17898;t;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 33371;Löningen;loningen;;;52.736526;7.7587838;;DE;Europe/Berlin;t;f;f;t;f;f;;;f;;f;;f;;f;;f;18258;t;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 33372;Mlawa;mlawa;;;53.1122632;20.3837208;;PL;Europe/Warsaw;t;f;f;t;f;f;;;f;;f;;f;;f;PLMAW;t;18568;t;;f;;f;5100219;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;Mielau;;;;;;;;;;;;;;Млава;;;姆瓦瓦
-33373;Montana;montana;;;43.25;23.14;;BG;Europe/Sofia;t;f;f;t;f;f;;;f;;f;;f;;f;BGMNT;t;18998;t;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
+33373;Montana;montana;;;43.25;23.14;;BG;Europe/Sofia;t;f;f;t;t;f;;;f;;f;;f;;f;BGMNT;t;18998;t;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
```
Comment (ConscritNeuneu) [inline stations.csv]: country_hint → true

File: stations.csv [hunk 7]
```diff
@@ -4798,7 +4798,6 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 6392;Montreux;montreux;8501300;85013003;46.4358640000;6.9104300000;;CH;Europe/Zurich;f;f;f;t;f;f;CHZJP;;t;;f;8501300;t;u0kbjj;t;CHZJP;t;16018;t;8501300;t;;f;8501300;f;;f;8501300;t;S01235;;;f;;f;8500116;f;;f;CHZJP;t;;f;f;;;;;;;;;;;;モントルー;몽트뢰;;;Монтрё;;Montrö;蒙特勒
 6393;Olten;olten;8500218;;47.3520090000;7.9076210000;;CH;Europe/Zurich;f;f;f;t;f;f;CHZJU;;t;;f;8500218;t;;f;CHZJU;t;19328;t;8500218;t;;f;8500218;f;;f;8500218;t;S01150;;;f;;f;;f;;f;CHZJU;t;;f;f;;;;;;;;;;;;オルテン;올텐;;;Ольтен;;;奧爾滕
 6395;Schaffhausen;schaffhausen;8503424;;47.6982830000;8.6327360000;;CH;Europe/Zurich;f;f;f;t;f;f;CHZKJ;;f;;f;8503424;t;u0qpk7;t;CHZKJ;t;12398;t;8503424;t;;f;8503424;f;;f;8503424;f;;;;f;;f;;f;;f;DESCN;t;;f;f;;;;;Schaffhouse;Sciaffusa;;;;;;シャフハウゼン;샤프하우젠;Szafuza;Sciaffusa;Шаффхаузен;;;沙夫豪森
-6396;Sierre;sierre;8585242;;;;;CH;Europe/Zurich;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;Siders;;;;;;;;;;シエール;;;;;;;谢尔
```
Comment (avandecreme) [inline stations.csv]: I thought we never deleted any station?

**Pull request merged**